### PR TITLE
Add animation video to sessions

### PR DESCRIPTION
**Pull request by beckykd**
request from @jyu00

### CODE REVIEW

File: docs/run/sessions.mdx [hunk 1]
```diff
@@ -33,6 +33,10 @@ The basic workflow for sessions is as follows:
 * The job must go through the normal queue to reactivate the session.
 </Admonition>
 
+<video title="A line of users is shown. The one in the current session becomes larger then the rest and has rotating arrows around them - indicating that they are the priority. Boxes (jobs) are seen being sent to the chandelier from the current user. Other users begin to pop up during the animation, and their jobs appear beneath them as they wait for the prioritized user to end their session. The prioritized user finishes and the lines moves ahead toward the system, so the next person's jobs can begin processing." class="max-w-auto h-auto" controls>
```
Comment (Eric-Arellano) [inline docs/run/sessions.mdx:36]: Nit

```suggestion
<video title="A line of users is shown. The one in the current session becomes larger then the rest and has rotating arrows around them, indicating that they are the priority. Boxes (jobs) are seen being sent to the chandelier from the current user. Other users begin to pop up during the animation, and their jobs appear beneath them as they wait for the prioritized user to end their session. The prioritized user finishes and the lines moves ahead toward the system, so the next person's jobs can begin processing." class="max-w-auto h-auto" controls>
```

**Review by Eric-Arellano — APPROVED** [review]
I love it! Great job with the alt text 🙌

**Comment by Eric-Arellano** [comment]
Check out the preview: https://qiskit-docs-preview-pr-622.1799mxdls7qz.us-south.codeengine.appdomain.cloud/run/sessions

File: docs/run/sessions.mdx [hunk 2]
```diff
@@ -33,6 +33,10 @@ The basic workflow for sessions is as follows:
 * The job must go through the normal queue to reactivate the session.
 </Admonition>
 
+<video title="A line of users is shown. The one in the current session becomes larger then the rest and has rotating arrows around them, indicating that they are the priority. Boxes (jobs) are seen being sent to the chandelier from the current user. Other users begin to pop up during the animation, and their jobs appear beneath them as they wait for the prioritized user to end their session. The prioritized user finishes and the lines moves ahead toward the system, so the next person's jobs can begin processing." class="max-w-auto h-auto" controls>
```
Comment (jyu00) [inline docs/run/sessions.mdx:36]: nit - can we move the animation to above line 32? So it's not sandwiched between two `Note` boxes.

Comment (jyu00) [inline docs/run/sessions.mdx:36]:
```suggestion
<video title="A line of users is shown. The one in the current session becomes larger than the rest and has rotating arrows around them, indicating that they are the priority. Boxes (jobs) are seen being sent to the QPU (represented by a chandelier) from the current user. Other users begin to pop up during the animation, and their jobs appear beneath them as they wait for the prioritized user to end their session. A progress bar pops above the prioritized user to represent the interactive TTL. Once the prioritized user finishes, the lines moves ahead toward the system, and the next person's jobs can begin processing." class="max-w-auto h-auto" controls>
```

**Pull request merged**